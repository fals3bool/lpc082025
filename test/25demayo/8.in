3
Ajo 2.00
Batata 2.00
Carne 2.00
2 2
Fideos 2.00
Pollo 2.00
