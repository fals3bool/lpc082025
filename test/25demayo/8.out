Ajo 4.0
Batata 4.0
Carne 4.0
